src/main/scala/mesosphere/marathon/core/launchqueue/impl/ReviveOffersActor.scala

```diff
@@ -75,7 +75,10 @@ class ReviveOffersActor(
         Resource.newBuilder.setName("disk")
           .setType(SCALAR)
           .setScalar(Scalar.newBuilder().setValue(resources.disk)))
-
+      .addResources(
+        Resource.newBuilder.setName("gpus")
+          .setType(SCALAR)
+          .setScalar(Scalar.newBuilder().setValue(resources.gpus)))
```

Comment:
shouldn't we add network bandwidth?

Reply:
I wasn't sure about the name of the network bandwidth resource in the protobuf message but I will add it before merging this PR